### ISSUE TITLE
Distributing a layer's output

### ISSUE THREAD

**Issue by ozancaglayan** [fchollet/keras#700]
Hi,

Is it possible to distribute a layer's output (say 1000 hidden units) into some sort of N sub-models composed of a non-linearity (say 500 units) followed by a softmax (say 20 classes) ?

This is a sequence-to-sequence feedforward architecture where inputs and outputs are sentences. So the overall N 20-class softmax probabilities are word probabilities in the target vocabulary.

I started to implement this overall architecture but got stock at this point. The closest thing seem the `Graph` container but I don't want different objectives for different paths.

Thanks!
